defaults/severities.yaml

```diff
@@ -95,7 +95,7 @@ TestFrameworkEvent: CRITICAL
 ElasticsearchEvent: ERROR
 SpotTerminationEvent: CRITICAL
 ScyllaRepoEvent: WARNING
-InfoEvent: NORMAL
+InfoEvent: ERROR
```

Comment:
Why is this needed?

Reply:
Since it's using this event with error severity

This file set the maximum severity for each type. (Not that I remember why we need it, i.e. one more way of filtering event I guess)

Reply:
The maximum severity?
Where do we use it?